doc/source/how-to/documenting.rst

```diff
@@ -72,13 +73,18 @@ editor to create reStructuredText (RST) files that you then place in
 :ref:`The \`\`doc\`\` directory`. The ``index.rst`` file in the ``doc/source`` directory
 defines the first level of your documentation hierarchy. The ``toctree``
 directive (which stands for "table of contents tree") indicates the maximum
-number of heading levels that the documentation is to display. Following this
-directive are the directory names for your documentation sections.
+number of heading levels that the documentation is to display in the right navigation
+pane. Following this directive are the directory names for your documentation sections.
 
 .. include:: diag/doc_layout.rst
 
-Each documentation section has its own ``index.rst`` file, as shown by the preceding
-figure. The following RST files provide examples of how to create ``index.rst`` files.
+As shown in the preceding figure, each documentation section has its own ``index.rst`` file.
+However, for search engine optimization, the names of the index files for sections should
```

Comment:
This is hard to describe given that we currently aren't following the SEO tips in the referenced section. I hope I've clearly conveyed what should be done versus what has been done!